include/cantera/thermo/ConstDensityTabulatedThermo.h

```diff
@@ -0,0 +1,113 @@
+/*
+ * ConstDensityTabulatedThermo.h
+ *
+ *  Created on: 04.06.2016

```

Comment:
Please do not add author information to each file; instead, add your name(s) to the `AUTHORS` file in the root of the repository.

Reply:
Fixed.